src/actors/hcd/neutral_fueling_actor.jl

```diff
@@ -0,0 +1,417 @@
+#= ====== =#
+#  PELLET  #
+#= ====== =#
+Base.@kwdef mutable struct FUSEparameters__ActorNeutralFueling{T<:Real} <: ParametersActor{T}
+    _parent::WeakRef = WeakRef(nothing)
+    _name::Symbol = :not_set
+    _time::Float64 = NaN
+    taupin::Entry{Float64} = Entry{Float64}("-", "Particle confinement time"; default=1.0)
+    twall::Entry{Float64} = Entry{Float64}("-", "Wall temperature (eV)"; default=10.0)
+    vwall::Entry{Float64} = Entry{Float64}("-", "Wall particle velocity (m/s)"; default=100.0)
+    model::Switch{Symbol} = Switch{Symbol}([:neucg, :none], "-", "Neutral gas fueling model"; default=:neucg)
+end
+
+mutable struct ActorNeutralFueling{D,P} <: SingleAbstractActor{D,P}
+    dd::IMAS.dd{D}
+    par::FUSEparameters__ActorNeutralFueling{P}
+    function ActorNeutralFueling(dd::IMAS.dd{D}, par::FUSEparameters__ActorNeutralFueling{P}; kw...) where {D<:Real,P<:Real}
+        logging_actor_init(ActorNeutralFueling)
+        par = par(kw...)
+        return new{D,P}(dd, par)
+    end
+end
+
+""" 
+    g0(x::Float64)::Float64
+
+    Interpolation of G function (Equation 12 in Burrel et al.) pulled from onetwo. Used where QuadGK calculation is slow
+"""
+
```

Comment:
```suggestion
```

docstring should not have a space between themselves and the function they are documenting